static/app/views/performance/landing/widgets/widgets/lineChartListWidget.tsx

```diff
@@ -630,11 +641,11 @@ export function LineChartListWidget(props: PerformanceWidgetProps) {
           );
         case PerformanceWidgetSetting.MOST_TIME_SPENT_DB_QUERIES:
         case PerformanceWidgetSetting.MOST_TIME_CONSUMING_RESOURCES:
-          const description: string = listItem[SpanMetricsField.SPAN_DESCRIPTION];
-          const group: string = listItem[SpanMetricsField.SPAN_GROUP];
-          const projectID: number = listItem['project.id'];
-          const timeSpentPercentage: number = listItem[fieldString];
-          const totalTime: number = listItem[`sum(${SpanMetricsField.SPAN_SELF_TIME})`];
+          const description = listItem[SpanMetricsField.SPAN_DESCRIPTION] as string;
+          const group = listItem[SpanMetricsField.SPAN_GROUP] as string;
+          const projectID = listItem['project.id'] as number;
+          const timeSpentPercentage = listItem[fieldString] as number;
+          const totalTime = listItem[`sum(${SpanMetricsField.SPAN_SELF_TIME})`] as number;
```

Comment:
Out of curiosity, why the change to `as` here?

Reply:
before the type was
`const totalTime: number = any` which was fine. But now it's `const totalTime = string | number` so it has to be casted to number. I could've left the `const totalTime: number` i guess

Reply:
K, just curious!